aries_cloudagent/protocols/didcomm_prefix.py

```diff
@@ -49,7 +49,7 @@ def qualify_all(cls, messages: dict) -> dict:
     def qualify_current(slug: str = None) -> str:
         """Qualify input slug with prefix currently in effect and separator."""
 
-        return qualify(slug, environ.get("DIDCOMM_PREFIX", DIDCommPrefix.NEW.value))
```

Comment:
(this was a tiny holdover from when we thought the default was going to be the new prefix - one day it will be again)